File: JavaNgArch/src/javangarch/mvc/home/LoginAttemptViewModel.java


package javangarch.mvc.home;

import javax.validation.constraints.NotNull;

import lombok.Getter;
import lombok.Setter;
import projectbase.mvc.DisplayName;
	@Getter@Setter
    @DisplayName("User")
    public class LoginAttemptViewModel
    {
    	@NotNull
    	@DisplayName("CorpCode")
    	private String corpCode;
    	@NotNull
        private String code;
        @NotNull
        private String password;


        public LoginAttemptViewModel()
        {
        	corpCode = "";
            code = "";
            password = "";
        }

    }

File: NetNgArch/src/NetNgArch.Web.Mvc/Home/LoginAttemptViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using ProjectBase.Web.Mvc;

namespace NetNgArch.Web.Mvc.Home
{
    [DisplayName("User")]
    public class LoginAttemptViewModel:BaseViewModel
    {
        [DisplayName("公司代码")]
        [Required]
        public string CorpCode { get; set; }
        [Required]
        public string Code { get; set; }
        [Required]
        public string Password { get; set; }

        public LoginAttemptViewModel()
        {
            CorpCode = "";
            Code = "";
            Password = "";
        }
    }
}
